.storybook/preview.js

```diff
@@ -30,6 +30,7 @@ export const globalTypes = {
     description: 'CSS Bedrock',
     defaultValue: 'enabled',
     toolbar: {
+      showName: false,
```

Comment:
Appears this dropped out of the documentation in the last couple of days. Probably in response to this deprecation: https://github.com/storybookjs/storybook/blob/362b3e941f6a96e2fb74e4495feb46657ecebc0e/code/addons/toolbars/src/components/ToolbarMenuList.tsx#L34-L40

Is there something that can be done with `title` (either empty string or `null`) to keep this as icon-only for now? If not, no big deal.

Reply:
Looks like an empty string did the trick.